validator/src/main/resources/schema/odf1_3.properties

```diff
@@ -25,10 +25,8 @@
 <entry key="manifest-schema">/schema/odf1.3/OpenDocument-manifest-schema-v1.3.rng</entry>
 <entry key="strict-schema"></entry>
 <entry key="schema">/schema/odf1.3/OpenDocument-schema-v1.3.rng</entry>
-<entry key="mathml-schema">/schema/mathml1.01/math.dtd</entry>
-<entry key="mathml2-schema">/schema/mathml2/mathml2.xsd</entry>
-<entry key="dsig-schema">/schema/odf1.3/OpenDocument-dsig-schema-v1.3.rng</entry>
-<entry key="path1"/>
-<entry key="filter"/>
+<entry key="mathml1.01-schema">/schema/mathml1.01/math.dtd</entry>
+<entry key="mathml3-schema">/schema/mathml3/mathml3.rng</entry>
+<entry key="dsig-schema">/schema/odf1.3/OpenDocument-dsig-schema-v1.3.rng</entry>   
```

Comment:
trailing whitespace was added to this line

Reply:
We should fix this in the root pom.xml where whitespacehandling is handled.
I fix this...